android/src/com/mapswithme/util/log/FileLoggerStrategy.java

```diff
@@ -18,6 +18,7 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.text.DateFormat;
+import java.text.SimpleDateFormat;
```

Comment:
Imports are managed by IDE, manual sorting is no needed. And it looks sorted, doesn't it?